crates/oxc_parser/src/ts/statement.rs

```diff
@@ -411,6 +413,7 @@ impl<'a> Parser<'a> {
                         .map(Declaration::FunctionDeclaration)
                 }
             }
+            // todo(yangchen): Handle other declarations with modifier (e.g., `abstract const a = 1`)
```

Comment:
Can you remove the todo comment and create an issue instead?

Reply:
Sure, fixed.